test_servise/test_clean.py

```
@@ -0,0 +1,53 @@
+    
+    print(f"Готово! Сохранено {len(result_data)} сообщений в {output_path}")
+
+# Пример запуска
+import os
+
+if __name__ == "__main__":
+        base_path = r"C:\Users\Alina\Desktop\chat-prep-etl\test_servise"
+        files = os.listdir(base_path)
+        target = "messages3_dump.json"
+        if target in files:
+            print(f"\nФайл '{target}' найден. Запускаю очистку...")
```

Ruff (0.15.12)

[warning] String contains ambiguous `а` (CYRILLIC SMALL LETTER A). Did you mean `a` (LATIN SMALL LETTER A)?

(RUF001)